voltdbclient/node_conn.go

```diff
@@ -301,6 +302,9 @@ func (nc *nodeConn) handleProcedureInvocation(pi *procedureInvocation) (int, err
 	EncodePI(encoder, pi)
 	n, err := nc.tcpConn.Write(encoder.Bytes())
 	if err != nil {
+		if strings.Contains(err.Error(), "write: broken pipe") {
```

Comment:
Properly a bit simpler for the client if we can refractor/standardize the error type here.

Reply:
Agreed. But that error comes from net package for broken pipe. What do you think will be agood error message?